qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/DependencyManager.groovy

```diff
@@ -264,7 +264,8 @@ class DependencyManager {
 
         CreateCustomerView createCustomerView
```

Comment:
do we need this variable if its a classvariable now?

Reply:
Nice Catch! I removed it 👍